airgun/entities/virtwho_configure.py

```diff
@@ -0,0 +1,120 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.virtwho_configure import (
+    VirtwhoConfigureCreateView,
+    VirtwhoConfigureEditView,
+    VirtwhoConfiguresView,
+    VirtwhoConfigureDetailsView,
+)
+
+
+class VirtwhoConfigureEntity(BaseEntity):
+
+    def reset_values(self, values):
```

Comment:
Nitpick: this should be named `_reset_values`, to signify private method.
We don't want anyone calling `session.virtwho_configure.reset_values()` from Robottelo :) (and it would be pointless)